core/lib/dal/src/blocks_dal.rs

```diff
@@ -883,6 +884,118 @@ impl BlocksDal<'_, '_> {
         Ok(())
     }
 
+    pub async fn save_l1_batch_inclusion_data(
```

Comment:
please add comment - as this does a lot more than just 'save' - is also checks that nothing has changed if data was already present.

This means that DA has to return 'consistent' inclusion data (which AFAIK is not mentioned in documentation)

Reply:
added and explained that